db/migrate/20220914095836_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :AuthorId
+      t.text :Title
+      t.text :Text
+      t.integer :CommentsCounter
+      t.integer :LikesCounter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- As mentioned above, kindly make sure Foreign key is included with the corresponding index 👍🏽